src/app/frontend/replicasetlist/replicasetlist_controller.js

```diff
@@ -28,10 +29,24 @@ export default class ReplicaSetListController {
    */
   constructor($state, replicaSets) {
     /** @export {!Array<backendApi.ReplicaSet>} */
-    this.replicaSets = replicaSets.replicaSets;
+    this.replicaSets = replicaSets.replicaSets || [];
```

Comment:
Ye, I've had to change this because of undefined error when I was checking length in `initState_()` method. I will try to change this in backend.